### ISSUE TITLE
bookmarking isotope child EIC

### ISSUE THREAD

**Issue by chubukov** [ElucidataInc/ElMaven#858]
**Describe the bug**
Bookmarking when displaying isotope child group does not do anything sensible

**To Reproduce**
Create a group with isotope children. Click on a child, and bookmark by double clicking or shift-dragging. You will get a new group where the "parent" has the m/z of the isotope, though the "C12 parent" has the correct parent m/z. And the only peaks present will be the ones where the original isotope is detected.

**Expected behavior**
I think that in most cases the desired behavior would be to modify the grouping/integration for that peakgroup. This behavior (modify peakgroup) really isn't present anywhere.

Perhaps a first step is to just disable this. Or create a new peakgroup with no compound, just mz@rt. Not sure.

**Comment by chubukov** [comment]
Bumping this, since it's actually a really frustrating experience to try to manually reintegrate isotopes -- you end up setting all the isotope filteres to degenerate values, deleting and re-bookmarking several times, and even that's sometimes insufficient. Would be great to be able to be able to in-place modify the peakgroup. #336 would make this mildly more palatable.

**Comment by shubhra-agrawal** [comment]
@chubukov Just so we're on the same page, in order to include isotopic peaks that might have been filtered out of a child group, you would like to modify isotopic peakgroups in-place on shift-drag integrating it from the Peak table (or double-click bookmarking based on the EIC grouping), right?

@shefalilathwal @sunil20dhakad, FYI.

**Comment by shefalilathwal** [comment]
@shubhra-agrawal yes that can be a good fix while we figure out a more suitable algorithmic solution to how to find isotopic peaks sensibly if unlabeled metabolite form is not present. 
We also run into this problem a lot with labeled data and have been modifying the data in post-processing to deal with the issue. As @chubukov pointed out, trying to manually integrate an isotopic peak behaves in an unexpected (and inaccurate) manner.

**Comment by chubukov** [comment]
@shubhra-agrawal sorry I missed your reply. I think that the in-place modification would be a pretty cool feature for all groups, not just isotopes. For parent groups of course you have the alternative to just delete and re-create (or merge from another table) but I can't think of a usable version of that for isotopes -- seems easier to enable the in-place modification.

**Comment by shubhra-agrawal** [comment]
@chubukov Great. Adding it to v0.9 then

@surbhipoddar FYI. Let's think about any UX challenges around this as well.
